documentation/changelog.rst

```diff
@@ -14,19 +14,11 @@ Infrastructure / Support
 ----------------------
 * Upgraded dependencies [see `PR #1847 <https://www.github.com/FlexMeasures/flexmeasures/pull/1847>`_]
 * Improve general description on SwaggerDocs page and add two settings for hosts: FLEXMEASURES_SUPPORT_PAGE and FLEXMEASURES_TOS_PAGE [see `PR #1851 <https://www.github.com/FlexMeasures/flexmeasures/pull/1851>`_]
+* Clean up cluttered base.html template [see `PR #1868 <https://www.github.com/FlexMeasures/flexmeasures/pull/1868>`_]
 
 Bugfixes
 -----------
-
-
-v0.30.2 | December XX, 2025
-============================
-
-Bugfixes
------------
-* Make ``soc-usage`` & ``soc-gain`` fields editable again in the UI [see `PR #1860 <https://www.github.com/FlexMeasures/flexmeasures/pull/1860>`_]
-* Prevent duplicate forecasts from being saved when re-running the forecasting pipeline within the same resolution window by preserving the non-floored ``belief_time`` using a new ``save_belief_time`` field [see `PR #1853 <https://github.com/FlexMeasures/flexmeasures/pull/1853>`_]
-
+* Fix broken array fields for flex-model modal dialogue [see `PR #1860 <https://www.github.com/FlexMeasures/flexmeasures/pull/1860>`_]
```

Comment:
That PR already has an entry, right? See https://github.com/FlexMeasures/flexmeasures/pull/1860/changes#diff-66ceabe4b6446d7912bb2f01ecccd5158b03d46b66f3aeb1170a681cd35dce3d